filestore/get.go

```diff
@@ -0,0 +1,85 @@
+package filestore
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	dag "github.com/ipfs/go-ipfs/merkledag"
+	pi "github.com/ipfs/go-ipfs/thirdparty/posinfo"
+	unixfs "github.com/ipfs/go-ipfs/unixfs"
+	cid "gx/ipfs/QmNp85zy9RLrQ5oQD4hPyS39ezrrXpcaa7R4Y9kxdWQLLQ/go-cid"
```

Comment:
I try to always leave a space between go-ipfs imports and gx imports

Reply:
Sorry.  Oversight, will fix.